src/components/Dropdown/Menu/Item/Item.tsx

```diff
@@ -1,63 +1,70 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import styles from './Item.module.css';
 
-export interface ItemProps {
-  className?: string;
+export type ButtonElementProps = React.ButtonHTMLAttributes<
+  HTMLButtonElement
+> & {
+  href?: undefined;
```

Comment:
hmm, I'm coming in without a lot of context here, but why are we creating Button and Div types that have `href` props?